README.md

```diff
@@ -126,41 +126,90 @@ helm delete my-galaxy
+| `service.port`                                   | Kubernetes service port                                                                                                                         |
+| `service.nodePort`                               | If `service.type` is set to `NodePort`, then this can be used to set the port at which Galaxy will be available on all nodes' IP addresses      |
+| `workflowHandlers.replicaCount`                  | The number of workflowHandlers to be spawned                                                                                                    |
+| `workflowHandlers.readinessProbe.enabled`        | Set to true to enable the Kubernetes readiness probe. A pod is *ready* when all of the containers in the pod are ready.                         |
+| `workflowHandlers.readinessProbe.periodSeconds`  | How frequently Kubernetes with check if the workflowHandlers are ready.                                                                         |
+| `workflowHandlers.readinessProbe.failureThreshold` | The number of times Kubernetes will retry the readiness probe before giving up.                                                                 |
+| `workflowHandlers.readinessProbe.timeoutSeconds` | How long Kubernetes will wait for a readiness probe to timeout.                                                                                 |
+| `workflowHandlers.livenessProbe.enabled`         | Set to `true` to enable Kubernetes liveness probes for workflowHandlers. Unresponsive handlers will be restarted by Kubernetes.                 |
+| `workflowHandlers.livenessProbe.periodSeconds`   | How frequently Kubernetes should check the responsiveness of workflowHandlers.                                                                  |
+| `workflowHandlers.livenessProbe.failureThreshold` | The number of times Kubernetes will try the liveness probe before giving up.                                                                    |
+| `workflowHandlers.livenessProbe.timeoutSeconds`  | How long Kubernetes will wait for a liveness probe to timeout.                                                                                  |
+| `webHandlers.replicaCount`                       | The number of replicas for the Galaxy web handlers                                                                                              |
+| `webHandlers.readinessProbe.enabled`             | Readiness probe configuration for the webHandlers                                                                                               |
+| `webHandlers.readinessProbe.periodSeconds`       | How frequently Kubernetes with check if the web handlers are ready.                                                                         |
+| `webHandlers.readinessProbe.failureThreshold`    |                                                                                                                                                 |
```

Comment:
I filled out most of the remaining descriptions, could you please either re-copy these or go back to the separate table as you had done in the previous PR?

Reply:
I was loathe to copy/paste that many descriptions so I went back to separate tables for the handlers and probes. I hope this is what you had in mind.